xDrip/Components/CustomTextField.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CustomTextField.swift
+//  xDrip
+//
+//  Created by Ivan Skoryk on 05.06.2020.
+//  Copyright © 2020 Faifly. All rights reserved.
+//
+
+import UIKit
+
+class CustomTextField: UITextField {
```

Comment:
Let's have a better name for it, e.g. `CustomInputTextField`. Also make it final